src/cpp/src/continuous_batching_impl.cpp

```diff
@@ -303,13 +307,20 @@ ContinuousBatchingPipeline::ContinuousBatchingImpl::generate(const std::vector<o
     bool continue_generation = true;
     while (has_non_finished_requests() && continue_generation) {
         try {
-            step();
+            const auto infer_start = std::chrono::steady_clock::now();
+            auto num_generated_tokens = step();
+            const auto infer_end = std::chrono::steady_clock::now();
+            const auto infer_ms = PerfMetrics::get_microsec(infer_end - infer_start);
+            raw_perf_counters.m_token_infer_durations.emplace_back(infer_ms);
+            // raw_perf_counters.m_inference_durations[0] += MicroSeconds(infer_ms);
```

Comment:
why is it commented out?

Reply:
I was no sure it's correct. Will return it batck

Reply:
Uncommented and ensured that is correctly calculated

Reply:
it's still commented